Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -1114,6 +1114,12 @@ theorem prod_ite_eq' [DecidableEq α] (s : Finset α) (a : α) (b : α → β) :
 #align finset.prod_ite_eq' Finset.prod_ite_eq'
 #align finset.sum_ite_eq' Finset.sum_ite_eq'
 
+@[to_additive (attr := simp)]
+theorem prod_ite_eq_iff [DecidableEq α] (s : Finset α) (a : α) (b : α → β)
```

Comment:
Hmm I'm not sure about this one, especially not as a simp lemma. I could just about believe that it's awkward to rewrite to get this by prod_ite_eq then h, but I worry that simp will try to prove `h` in cases where it's a big time sink